auth_saml/models/res_users.py

```diff
@@ -0,0 +1,243 @@
+# Copyright (C) 2010-2016 XCG Consulting <http://odoo.consulting>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+import logging
+import passlib
+
+from odoo import api, fields, models, _, SUPERUSER_ID
+from odoo.exceptions import ValidationError, AccessDenied
+
+_logger = logging.getLogger(__name__)
+
+try:
+    import lasso
+except ImportError:
+    _logger.debug('Cannot `import lasso`.')
```

Comment:
```suggestion
import lasso
```

No need to protect imports in v12+.